### PR TITLE
Replace disconnect and system info symbols for agents

### PR DESCRIPTION
**Pull request by mawinter69**
The symbol to disconnect was the same as that to delete the agent. Add a new symbol that better fits for disconnecting (There is no fitting icon at ionicons, so this is one from tabler-icons that I modified to have a bigger viewport).
The symbol for System Information was still on old colored svg. Replace this with information-circle from ionicons

After:
image

### Testing done

Added an inbound agent from a Jenkins build started with this change and confirmed that the new icons were displayed as expected in Google Chrome, Microsoft Edge, and Mozilla Firefox.

No test automation because we don't automate testing of icon changes.

### Proposed changelog entries

- Replace disconnect and system info symbols for agents.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [x] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [x] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [x] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [x] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

N/A

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8015"></a>

### CODE REVIEW

**Comment by MarkEWaite** [comment]
Thanks for the pull request @mawinter69 !  I've updated the description with the testing that I performed.  

It helps the review process very much if the submitter completes the "Testing done" section to indicate the testing that they have performed.  It also helps the review process if the submitter marks the submitter checklist items as "completed" by replacing the "[ ]" text at the start of the checklist item with "[x]".  Thanks again

**Review by MarkEWaite — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
Thanks a lot!

---

/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
